src/main/kotlin/no/nav/helse/sporenstreks/web/dto/RefusjonskravDto.kt

```diff
@@ -43,6 +43,7 @@ data class RefusjonskravDto(
             if (!tariffEndring) {
                 // kan ikke kreve refusjon for dager før tre måneder siden
```

Comment:
kommentaren blir vel da:
kan ikke kreve refusjon for dager før tre måneder (minus 15 dager) siden